assignment-client/src/avatars/AvatarMixerClientData.h

```diff
@@ -45,6 +45,7 @@ class AvatarMixerClientData : public NodeData {
 
     int parseData(ReceivedMessage& message) override;
     AvatarData& getAvatar() { return *_avatar; }
+    const AvatarData& getAvatar() const { return *_avatar; }
```

Comment:
Needed this call for `computeBubbleBox()` helper function in `AvatarMiserSlave.cpp`.